pkg/apis/stats/v1alpha1/types.go

```diff
@@ -146,3 +146,43 @@ type RuleTrafficStats struct {
 	Name         string       `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
 	TrafficStats TrafficStats `json:"trafficStats,omitempty" protobuf:"bytes,2,opt,name=trafficStats"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +resourceName=nodeiplatencystats
+// +genclient:onlyVerbs=create
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeIPLatencyStat contains the latency stat of a Node.
+type NodeIPLatencyStat struct {
+	metav1.TypeMeta
+	metav1.ObjectMeta `protobuf:"bytes,1,opt,name=objectMeta"`
+
+	// The list of NodeIPLatency.
+	NodeIPLatencyList []NodeIPLatencyEntry `protobuf:"bytes,2,rep,name=nodeIPLatencyList"`
+}
+
+// NodeIPLatencyEntry contains the latency stats of a Node.
+type NodeIPLatencyEntry struct {
+	// The Node's name.
+	NodeName string `protobuf:"bytes,1,opt,name=nodeName"`
```

Comment:
it's surprising that you don't have `json` tags. Can you check `AntreaClusterNetworkPolicyStats` (same file) for reference.

Reply:
Ok, I migrated these codes from another `types.go` before, and the other types.go didn't have a json tag, I will update it by this file.